dashboard/app/controllers/aichat_controller.rb

```diff
@@ -63,6 +63,47 @@ def log_chat_event
     render(status: :ok, json: response_body)
   end
 
+  # params are studentUserId: number, currentLevelId: number, scriptId: number, (optional) scriptLevelId: number
```

Comment:
super minor: maybe just `levelId` instead of `currentLevelId`, since we're fetching chat history for some given level ID (the server doesn't necessarily need to know if it's the "current" one or not)? That way it's also consistent with `scriptId` and `scriptLevelId`.